packages/core/package.json

```diff
@@ -27,5 +27,10 @@
     "mini-css-extract-plugin": "^2.4.2",
     "webpack": "^5.56.0",
     "webpack-cli": "^4.8.0"
+  },
+  "peerDependencies": {
+    "@department-of-veterans-affairs/formation": "*",
+    "react": "^17",
+    "react-dom": "*"
```

Comment:
A note for the future from a commit message:

> Once we are no longer using the following dependencies we shouldn't need
> to declare these peers:
>
> - react-focus-on          (Modal)
> - react-scroll            (CollapsiblePanel)
> - react-transition-group  (ExpandingGroup)

Reply:
Related Tech Debt Tickets:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/34436
https://github.com/department-of-veterans-affairs/va.gov-team/issues/34437
https://github.com/department-of-veterans-affairs/va.gov-team/issues/34438